### PR TITLE
CARBONDATA-17 - select count(*) from table where column_x = 'value' is not returning the correct count

### PR DESCRIPTION
**Pull request by mohammadshahidkhan**
CARBONDATA-17 - select count(*) from table where column_x = 'value' is not returning the correct count if the first measure is having all values null.

### CODE REVIEW

**Comment by gvramana** [comment]
retest this please

**Comment by kumarvishal09** [comment]
retest this please

File: processing/src/main/java/org/carbondata/processing/graphgenerator/GraphGenerator.java [hunk 1]
```diff
@@ -472,7 +472,8 @@ private StepMeta getSliceMeregerStep(GraphConfigurationInfo configurationInfo,
       String[] aggType = configurationInfo.getAggType();
       StringBuilder builder = new StringBuilder();
       for (int i = 0; i < aggType.length - 1; i++) {
-        if (aggType[i].equals(CarbonCommonConstants.COUNT)) {
```
Comment (kumarvishal09) [inline processing/src/main/java/org/carbondata/processing/graphgenerator/GraphGenerator.java]: no need to change here as this is data loading flow and count star with filter for query execution

**Comment by kumarvishal09** [comment]
LGTM , except one small comment.

File: processing/src/main/java/org/carbondata/processing/graphgenerator/GraphGenerator.java (hunk 1 above)
Comment (mohammadshahidkhan) [inline processing/src/main/java/org/carbondata/processing/graphgenerator/GraphGenerator.java]: fixed

**Pull request merged**